### PR TITLE
Move Index to Indexer package

### PR DESCRIPTION
**Pull request by dmosites**

### CODE REVIEW

File: source/indexer/reports/Indexer.md [hunk 1]
```diff
@@ -13,13 +13,11 @@ The Swap Protocol is a peer-to-peer protocol for trading Ethereum tokens that al
 
 Indexer is comprised of several contracts.
 
-[@airswap/index/contracts/interfaces/ILocatorWhitelist.sol](../contracts/ILocatorWhitelist.sol) @ [b87d292aaf6e28ede564b7ea28ece39219994607](https://github.com/airswap/airswap-protocols/commit/b87d292aaf6e28ede564b7ea28ece39219994607)
+[@airswap/indexer/contracts/interfaces/ILocatorWhitelist.sol](../contracts/ILocatorWhitelist.sol) @ [b87d292aaf6e28ede564b7ea28ece39219994607](https://github.com/airswap/airswap-protocols/commit/b87d292aaf6e28ede564b7ea28ece39219994607)
 
-[@airswap/index/contracts/interfacts/IIndexer.sol](../contracts/IIndexer.sol) @ [b87d292aaf6e28ede564b7ea28ece39219994607](https://github.com/airswap/airswap-protocols/commit/b87d292aaf6e28ede564b7ea28ece39219994607)
+[@airswap/indexer/contracts/interfacts/IIndexer.sol](../contracts/IIndexer.sol) @ [b87d292aaf6e28ede564b7ea28ece39219994607](https://github.com/airswap/airswap-protocols/commit/b87d292aaf6e28ede564b7ea28ece39219994607)
 
-[@airswap/index/contracts/Index.sol](../contracts/Index.sol) @ [b87d292aaf6e28ede564b7ea28ece39219994607](https://github.com/airswap/airswap-protocols/commit/b87d292aaf6e28ede564b7ea28ece39219994607)
```
Comment (alicevhenshaw) [inline source/indexer/reports/Indexer.md]: Why have you removed the Index.sol contract from the list of `Indexer is comprised of several contracts.`?

Comment (dmosites) [reply source/indexer/reports/Indexer.md]: That was actually out of place beforehand. The Index has its own report in the same folder.

File: yarn.lock [hunk 2]
```diff
@@ -2,6 +2,19 @@
 # yarn lockfile v1
 
 
+"@airswap/order-utils@0.3.13":
```
Comment (alicevhenshaw) [inline yarn.lock]: I dont think this should've been pushed. If i run `yarn` locally, this gets deleted.

File: source/indexer/reports/Indexer.md (hunk 1 above)
Comment (alicevhenshaw) [reply source/indexer/reports/Indexer.md]: I see. But the Indexer is still comprised of the index isnt it?

File: yarn.lock (hunk 2 above)
Comment (dmosites) [reply yarn.lock]: That's right. The pre-swap-check had an old reference, which I've updated. Will update the lock file.

File: source/indexer/reports/Indexer.md (hunk 1 above)
Comment (dmosites) [reply source/indexer/reports/Indexer.md]: Sure. Added

**Pull request merged**